testdata/dnn/tensorflow/generate_tf_models.py

```diff
@@ -66,12 +80,36 @@ def runModel(inpName, outName, name):
         writeBlob(inputData, name + '_in')
         writeBlob(outputData, name + '_out')
 
-def save(inp, out, name, quantize=False, optimize=True):
+
+def save(inp, out, group_name, name=None, quantize=False, optimize=True, is_gen_data=True):
```

Comment:
Seems like we have behavior duplication with `name` and `is_gen_data`. If we have `name not None` we reuse input data, so why not make `is_gen_data` internal?

Reply:
`is_gen_data` may be useful for those cases, when, for example, we've decided to slightly modify net architecture in test group and we don't need to regenerate input data, so `name=None`, `is_gen_data=False`. In such case, if we regenerate input data, we will need to rerun all other cases inside this test group for inputs-outputs matching. 
Do we need to support this case? If not, I'll remove `is_gen_data`.

Reply:
Ok, I got it, thanks!